### ISSUE TITLE
0.2.1 from extras-devel (n900) starts but only shows black screen

### ISSUE THREAD

**Issue by kotc** [mkiol/Cyklop#2]
title is self explanatory. running cssu-thumb. is project alive and is there a chance for 0.3+ release?

**Comment by mkiol** [comment]
No, unfortunately I can't support this project. I don't have N900 or N9 any more.

To be frank, I'm quite surprise that someone is trying to use Cyklop these days!

**Comment by smyhu** [comment]
You would be surprised how many people were using it. Actually it stopped updating quite recently, just after I discovered it supports my fav city (Stalowa Wola). Anyway, it was a pleasure to use it. Greetings
